src/lib/io/sol-gpio-impl-contiki.c

```diff
@@ -99,7 +99,7 @@ sol_gpio_open_raw(uint32_t pin, const struct sol_gpio_config *config)
             return NULL;
         }
     } else {
-        if (pin < 0 || pin > 7) {
+        if (pin > 7) {
```

Comment:
On Mon, 18 Apr 2016 11:10:10 -0700, Gustavo Lima Chaves wrote:
> > @@ -99,7 +99,7 @@ sol_gpio_open_raw(uint32_t pin, const struct sol_gpio_config *config)
> >              return NULL;
> >          }
> >      } else {
> > -        if (pin < 0 || pin > 7) {
> > +        if (pin > 7) {
> 
> I'm curious on why < 0 would be valid

It wouldn't, but pin is unsigned.